Auth/class.LDAPAuthenticator.php

```diff
@@ -190,11 +203,13 @@ public function login($username, $password)
         }
         $user = $user[0];
         $server->unbind();
+
         $ret = $server->bind($user->dn, $password);
         if($ret !== false)
         {
-            return array('res'=>true, 'extended'=>$user); 
+            return array('res'=>true, 'extended'=>$user);
         }
+        $server->unbind();
```

Comment:
I'm not sure why we wouldn't want to act as that user with the LDAP server for the rest of the transaction... What conflicts do you see?